chainer/dataset/convert.py

```diff
@@ -106,6 +106,14 @@ def converter():
     If the batch were requested to be converted to ChainerX with such
     converters, :class:`RuntimeError` will be raised.
 
+    .. note:: Not compatible with pickle
```

Comment:
The syntax of directives is similar to block quotes.  Indent the paragraphs by 3 or 4 spaces, like
```
.. note::

    Not compatible with pickle.

    foo
    bar
```

Example: https://github.com/cupy/cupy/blob/a98f05a7ae9fb76c50672b2aac6f5dd5b2c62bf3/cupy/core/core.pyx#L1203-L1249 to generate [`cupy.ndarray.html#cupy.ndarray.__setitem__`](https://docs-cupy.chainer.org/en/latest/reference/generated/cupy.ndarray.html#cupy.ndarray.__setitem__)